monitor/subscription.go

```diff
@@ -247,53 +265,74 @@ func (s *Subscription) AddNodes(nodes ...string) error {
 
 // AddNodeIDs adds nodes
 func (s *Subscription) AddNodeIDs(nodes ...*ua.NodeID) error {
+
+	requests := make([]Request, len(nodes))
+
+	for _, node := range nodes {
+		requests = append(requests, Request{
```

Comment:
This appends _after_ the pre-allocated nodes from L269. Either init an empty slice, or (as I prefer), use the index from `range` to set each element.

see: https://play.golang.org/p/PrCpdwROEdF

Reply:
Changed to used the index as suggested.